stack.py

```diff
@@ -57,22 +57,77 @@ def parse_program(code):
     [Instr('push', [1.0])]
     >>> list(parse_program('↔'))
     [Instr('swap')]
+
+    Prefixes are placed before the instruction
+    Some prefixes also have sigils
+    >>> list(parse_program('♯ +'))
+    [Instr('add', [], ['quiet'])]
     """
-    for line in re.split('\n|;', code):
+    split_program = re.split('\n|;', code)
+    label_indexes = get_label_indexes(split_program);
```

Comment:
That would make forward label jumps much harder though. 

`
push 0
jump @label
@label
push 1
push 2
`

Would require storing unknown labels and matching them with future labels. I also think that label processing should definitely be done before parsing since labels aren't really instructions or arguements.